packages/broker/src/plugins/operator/MaintainTopologyService.ts

```diff
@@ -16,72 +13,49 @@ function toStreamIDSafe(input: string): StreamID | undefined {
     }
 }
 
-function singletonSet<T>(element: T): Set<T> {
-    return new Set<T>([element])
-}
-
 export class MaintainTopologyService {
     private readonly streamrClient: StreamrClient
-    private readonly operatorClient: OperatorClient
+    private readonly maintainTopologyHelper: MaintainTopologyHelper
     private readonly subscriptions = new Multimap<StreamID, Subscription>()
     private readonly synchronizer = new SetMembershipSynchronizer<StreamID>()
     private readonly concurrencyLimit = pLimit(1)
+    private readonly logger: Logger
 
-    constructor(streamrClient: StreamrClient, operatorClient: OperatorClient) {
+    constructor(streamrClient: StreamrClient, maintainTopologyHelper: MaintainTopologyHelper, logger: Logger) {
         this.streamrClient = streamrClient
-        this.operatorClient = operatorClient
+        this.maintainTopologyHelper = maintainTopologyHelper
+        this.logger = logger
     }
 
     async start(): Promise<void> {
-        this.operatorClient.on('addStakedStream', this.onAddStakedStream)
-        this.operatorClient.on('removeStakedStream', this.onRemoveStakedStream)
-        await this.operatorClient.start()
-        const rawStreamIds = await this.operatorClient.getStakedStreams()
-        const blockNumber = 10 // TODO: fix properly
-        const streamIds = new Set(compact([...rawStreamIds].map(toStreamIDSafe)))
-        const { added } = this.synchronizer.ingestSnapshot(streamIds, blockNumber)
-        for (const streamId of added) {
-            await this.addStream(streamId, blockNumber)
-        }
-        logger.info('Started')
+        this.logger.info('Starting MaintainTopologyService')
+        this.maintainTopologyHelper.on('addStakedStream', this.onAddStakedStreams)
+        this.maintainTopologyHelper.on('removeStakedStream', this.onRemoveStakedStream)
+        await this.maintainTopologyHelper.start()
+        this.logger.info('Started MaintainTopologyService')
     }
 
-    // eslint-disable-next-line class-methods-use-this
     async stop(): Promise<void> {
-        this.operatorClient.stop()
-        logger.info('stopped')
+        this.maintainTopologyHelper.stop()
+        this.logger.info('stopped')
```

Comment:
```suggestion
        this.logger.info('Stopped')
```
Capitalize in accordance with https://github.com/streamr-dev/network/blob/main/internal-docs/logging-practices.md